src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -271,6 +281,44 @@ private bool ProcessAndValidateArguments()
         if (!Directory.Exists(IntermediateOutputPath))
             Directory.CreateDirectory(IntermediateOutputPath);
 
+        if (!string.IsNullOrEmpty(NetTracePath))
+        {
+            if (!File.Exists(NetTracePath))
+            {
+                Log.LogError($"NetTracePath={NetTracePath} doesn't exist");
+                return false;
+            }
+            if (!File.Exists(PgoBinaryPath))
+            {
+                Log.LogError($"NetTracePath was provided, but {nameof(PgoBinaryPath)}='{PgoBinaryPath}' doesn't exist");
+                return false;
+            }
+
+            var OutputMibcPath = Path.Combine(OutputDir, Path.ChangeExtension(Path.GetFileName($"{NetTracePath}"), ".mibc"));
```

Comment:
Thanks. I think creating the `.mibc` file in the `OutputDir` is fine. Not sure if theres a desire for it to be in a particular place. I think it would only be grabbed for debugging purposes in case the resulting `.mibc` from the `.nettrace` contains unwanted methods for AOT.